qdrant-landing/content/blog/enterprise-vector-search-2025.md

```diff
@@ -0,0 +1,100 @@
+
+### 3. Observability & Monitoring: Understanding Performance at Scale
+
+AI infrastructure is a **black box without real-time metrics**. Teams need **monitoring** to **track latency, resource usage, and query performance**, avoiding costly downtime and slow responses. 
+
+### 4. Automation & Scalability: Reducing Manual Overhead
+
+Deploying and managing clusters manually is **slow, error-prone, and expensive**. Without **APIs**, enterprises waste time on configuration instead of **innovation**. 
+
+Without solving these, enterprises risk **security breaches, compliance failures, downtime, and operational inefficiencies**. That's why we built a feature suite designed to **secure, simplify, and scale vector search**.
+
+## Meet Qdrant's Enterprise Feature Suite
+
+Our enterprise solution handles scaling and securing vector search for production AI systems. Here's how:
+
+### Enterprise-Grade Security & Access Control
+
+**Cloud RBAC (Role-Based Access Control)** ***(Coming soon)*** → Define **granular permissions** to manage clusters, billing and Hybrid Cloud deployments.  
+[**Single Sign-On (SSO) Integration**](https://qdrant.tech/documentation/cloud/qdrant-cloud-setup/#enterprise-single-sign-on-sso) → Streamlined authentication via:  
+- Okta   
+- Entra ID (formerly Azure Active Directory)  
+- Azure Active Directory  
+- Active Directory/LDAP  
+- Azure Active Directory Native  
+- ADFS  
+- Google Workspace  
+- OpenID Connect  
+- PingFederate  
+- SAML  
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EtUcA-MCZJM?si=RT2qMkA1pzbIWZA5" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+- [**Database API Keys with Granular Access Control**](https://qdrant.tech/documentation/cloud/authentication/) → Restrict API key access **per cluster and per collection**, enforce expirations, and revoke credentials instantly.
+
+**Why it matters:** Security is **baked in, not bolted on**. So you get **centralized control** over access management—simplifying compliance and reducing risk.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/3c-8tcBIVdQ?si=Kh_tcmTVGjPnq7Zh" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe> 
+
+### Full Observability with Monitoring
+
+* [**Monitoring**](https://qdrant.tech/documentation/guides/monitoring/) → **Real-time metrics via Prometheus/OpenMetrics**, integrating directly into your existing monitoring stack (Datadog, Grafana, etc.).
```

Comment:
I think we should rather link to https://qdrant.tech/documentation/cloud/cluster-monitoring/